### ISSUE TITLE
question about loss function

### ISSUE THREAD

**Issue by haimingZZ** [xiaohu2015/DeepLearning_tutorials#5]
I have one question. Why  don't  i see the loss function for training in your code?

**Comment by xiaohu2015** [comment]
@haimingZZ sorry， the training code is not included for some models
